ts/dx.all.d.ts

```diff
@@ -2975,6 +2975,42 @@ declare module DevExpress.fx {
      */
     export function stop(element: Element, jumpToEnd: boolean): void;
 }
+declare module DevExpress.pdfExporter {
+    /**
+     * [descr:PdfExportBaseProps]
+     */
+    export interface PdfExportBaseProps {
```

Comment:
Another excessive split to Base and Grid props.

Reply:
It is autogenerated part of the merge after update-ts was called.